libs/core/src/integration/outbox.schema.ts

```diff
@@ -125,6 +125,18 @@ export const Outbox = z.union([
       event_payload: events.FarcasterVoteCreated,
     })
     .merge(BaseOutboxProperties),
+  z
+    .object({
```

Comment:
why are you adding these two to the outbox?

Reply:
throws a type error if you don't

Reply:
is it b/c you are adding them to the pairs?